eluthia/zipapp_script.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+import subprocess
+import zipfile
+import os
+import time
+
+APP_PATH = os.getcwd() # We actually have to use this instead of abspath to get the correct value
+
+HISTORY_PATH = '/var/lib/badtrack/history' #TODO: use environment variable.
+UNPACK_PATH = f'{APP_PATH}/zip_unpack' #TODO: This could also be an environment variable?
+NEW_DEB_PATH = f'{UNPACK_PATH}/new_debs'
+OLD_DEB_PATH = f'{UNPACK_PATH}/old_debs'
+
+
+def get_version(package_name):
+    """
+    Verifies that a package is installed, and get it's version.
+    Input:
+        string package_name - the name of the package that should be checked
+    Output:
+        string version - version of the package
+            NoneType None denotes that the package is not installed.
+    """
+    try:
+        # Get package version number by parsing dpkg -s.
+        return str(subprocess.check_output(['dpkg', '-s', package_name]))\
```

Comment:
```py
try:
  # Limit amount of code to catch
   output = subprocess.check_output(['dpkg', '-s', package_name])
except CallProcessError:
   return None

_, version = str(output).split('Version: ')
return version.strip()
```
Any argument for 'dpkg' which gives you even more specific output that just outputs version number?